test/test_runners.rb

```diff
@@ -18,9 +18,9 @@ def test_new
       state=nil
       assert_nothing_raised() { state=runner.run(spec) }
       assert_equal(1, state["steps"].size)
-      assert_equal(4, queue.size)
+      assert_equal(6, queue.size)
```

Comment:
One additional message stems forth from the missing queue push in the `Messaging` module the other message stems forth from the additional message in the `run_scenario` method.